src/bootstrap/config_processor.rs

```diff
@@ -38,8 +38,7 @@ where
     let api_key_model = ApiKeyRepoModel::new(
         "default".to_string(),
```

Comment:
delete api key logic might check this string and prevent deletion.

Use case:
- redis mode is used
- api key got stored 
- user restarted instance and provided different value for API_KEY env var.
Current validation would allow deletion in this case as it compares env var.
Not sure how much valid is above case but anyway, it could produce side effects